Framework.Core/System/Runtime/CompilerServices/SetsRequiredMembersAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿// https://github.com/dotnet/runtime/blob/v7.0.13/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/SetsRequiredMembersAttribute.cs
+#if !NET7_0_OR_GREATER
+#define SYSTEM_PRIVATE_CORELIB
```

Comment:
Hmm actually, I removed these from the ported files.
Probably it is a better idea simply to have a copy-paste and define this globally.

👍